TEMPLATE.md

```diff
@@ -1,4 +1,7 @@
-# <Name of Feature>
+# {Name of Feature}
+
+- Owner: {Your Name} {you@example.com}, {@you} on the Elm Slack
```

Comment:
Hey, no worries... I see what you're going for with these words, but I was imagining that these documents would be written by the community and maintained collaboratively, without a designated steward etc. *Owner* in particular seems to imply "this is mine, don't touch".